elastica/boundary_conditions.py

```diff
@@ -294,6 +299,12 @@ def __init__(self, *fixed_data, **kwargs):
             Tuple of position-indices that will be constrained
         constrained_director_idx : tuple
             Tuple of director-indices that will be constrained
+        translational_constraint_selector: np.array = np.array([True, True, True])
+            np.array of type bool indicating which translational degrees of freedom (dof) to constrain.
+            If entry is True, the corresponding dof will be constrained.
+        rotational_constraint_selector: np.array = np.array([True, True, True])
```

Comment:
ditto^

Reply:
Fixed in https://github.com/GazzolaLab/PyElastica/pull/143/commits/03fc1502110db67904ceb26648b378d982e6cdf1